synapse/rest/key/v2/remote_key_resource.py

```diff
@@ -157,14 +158,22 @@ async def query_keys(
     ) -> JsonDict:
         logger.info("Handling query for keys %r", query)
 
-        store_queries = []
+        cached: Dict[Tuple[str, str], Optional[FetchKeyResultForRemote]] = {}
```

Comment:
Not following this commit so much. I guess this is going to be replaced with some kind of cache from the DB in the future?

Reply:
Oh, I think the original variable name is just confusing. Maybe just call this `server_keys` instead of `cached`?

Reply:
Yeah, its "cached" in the sense that the server already has those keys in its DB